components/blitz/resources/omero/Constants.ice

```diff
@@ -214,14 +214,6 @@ module omero {
      **/
     module analysis {
 
-       /** namespaces related to the FLIM analysis. **/
```

Comment:
Sorry I misunderstood your comment, It showed up a "global comment", we can certainly mark them deprecated in the constants.ice and in the pojo class.
And clean up elsewhere